README.md

```diff
@@ -40,31 +40,28 @@ The class components in the current version are `NumberGuessingGame` and `GuessC
 If you want to attempt this on your own without a step by step walkthrough first then leave the section below collapsed.
 
 ## Step by Step
-<details>
-    <summary>Expand to see step by step actions to convert to function components with hooks.</summary>
```

Comment:
I like the idea of an accordion here, but I didn't notice it at first and I'm worried students won't either

Reply:
I want to give this a try for this lesson as one of the bits of feedback in my last session was about how the instructions were just step by step walkthroughs. I will tweak the wording though to say 'Click the arrow to expand...' which should provide more explicit instructions for taking this action.